src/test/java/seedu/address/model/person/RatingTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class RatingTest {
+    @Test
+    public void constructor_invalidRating_throwsIllegalArgumentException() {
+        int invalidRating = -1;
+        assertThrows(IllegalArgumentException.class, () -> new Rating(invalidRating));
+    }
+
+    @Test
+    public void setRating_invalidRating_throwsIllegalArgumentException() {
+        Rating testRating = new Rating(0);
+        assertThrows(IllegalArgumentException.class, () -> testRating.setRating(-1));
+    }
+
+    @Test
+    public void isValidRating() {
+        // invalid addresses
+        assertFalse(Rating.isValidRating(124)); // empty string
```

Comment:
modify the comment from "addresses" to "rating"

Reply:
Fixed!